content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,60 @@
+---
+reviewers:
+- derekwaynecarr
+title: Process ID Limits And Reservations
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+Kubernetes allow you to limit the number of process IDs (PIDs) that a {{< glossary_tooltip term_id="Pod" text="Pod" >}} can use. You can also reserve a number of allocatable PIDs for each {{< glossary_tooltip term_id="node" text="node" >}} for use by the operating system and daemons (rather than by Pods).
+
+<!-- body -->
+
+Process IDs (PIDs) are a fundamental resource on nodes. It is trivial to hit the task limit without hitting any other resource limits, which can then cause instability to a host machine.
+
+Cluster administrators require mechanisms to ensure that Pods running in the cluster cannot induce PID exhaustion that prevents host daemons (such as the kubelet or kube-proxy, and potentially also the container runtime) from running. In addition, it is important to ensure that PIDs are limited among Pods in order to ensure they have limited impact on other workloads on the same node.
+
+{{< note >}}
+On Linux, consider raising `/proc/sys/kernel/pid_max` if your operating system sets this to a low default, such as `32768`.
+{{< /note >}}
+
+You can configure a kubelet to limit the number of PIDs a given pod can consume. For example, if your node's host OS is set to use a maximum of `262144` PIDs and `250` pods, one can give each pod a budget of `1000` PIDs to prevent using up that node's overall number of available PIDs. If the admin wants to overcommit PIDs similar to CPU or memory, they may do so as well with some additional risks. Either way, no one pod can bring the whole machine down. This kind of resource limiting helps to protect against simple fork bombs from affecting service cluster-wide.
```

Comment:
Can we break the long lines so that we can comment on a specific phrase or word?
In the future, this kind of long lines are difficult to change, diff compare.

Reply:
> if your node's host OS is set to use a maximum of `262144` PIDs and `250` pods

This expression is is confusing. How am I supposed to set the maximum Pods on the host OS?
If I'm understanding this correctly, pods limits can be set via some command line flags, some configuration files or even resource quota.
But all these settings are related to kubernetes control plane, not the host OS.

Reply:
> Either way, no one pod can bring the whole machine down.

Do you mean "no one pod should be allowed to bring the whole machine down", or "no one pod is able to bring the whole machine down"?

Reply:
> This kind of resource limiting helps to protect against simple fork bombs from affecting service cluster-wide.

This kind of resource limiting helps to protect cluster-wide services against simple fork bombs.

or 

This kind of resource limiting helps to prevent simple fork bombs from affecting service cluster-wide.

Leaving this to a native speaker.

Reply:
> Can we break the long lines so that we can comment on a specific phrase or word?

Definitely. I didn't know what's the best practice here.

Reply:
rephrased all